docs/index.md

```diff
@@ -53,6 +55,13 @@ You can now checkout the included dashboards, e.g. the [Cluster Monitoring Overv
 
 ![Grafana Import Dashboard](grafana_cluster_overview.png)
 
+_Note:_ If persistent storage is not set up in your cluster, the preset datasource and dashboards will vanish if the Grafana Pod gets rescheduled. To get them back run:
+
+```bash
```

Comment:
`bash` for highlighting is only useful when displaying a bash script, where keywords like `for`, `do` and `done` should be highlighted. For shell commands like this I would always recommend `nohighlight`.